professor2.spec

```diff
@@ -22,13 +22,22 @@ sed -i -e 's|-std=c[+][+]11|-std=c++%{cms_cxx_standard}|' pyext/setup.py
 # Same for Makefile
 grep -q 'CXXSTD := c[+][+]11' Makefile
 sed -i -e 's|CXXSTD := c[+][+]11|CXXSTD := c++%{cms_cxx_standard}|' Makefile
+sed -i -e "s|pip install -vv|pip install --target ../${PYTHON3_LIB_SITE_PACKAGES} -vv|" Makefile
 
-%define build_flags CPPFLAGS=-I${EIGEN_ROOT}/include/eigen3 PYTHON=$(which python3) PROF_VERSION=%{realversion} PYTHONPATH=./${PYTHON3_LIB_SITE_PACKAGES}
+# Update Python2 to Python3
+for i in `ls bin`
+do
+   echo "bin/${i}"
+   sed -i -e 's|/usr/bin/env python|/usr/bin/env python3|' "bin/${i}"
+done
 
+%define build_flags CPPFLAGS=-I${EIGEN_ROOT}/include/eigen3 PYTHON=$(which python3) PROF_VERSION=%{realversion} PYTHONPATH=./${PYTHON3_LIB_SITE_PACKAGES}
 %build
-make %{build_flags}
+make PREFIX=%{i} %{build_flags}
```

Comment:
@DickyChant , I do not remember why we are not building `professor2` with `-j N` option. Do you know if there is any issue building it in parallel?

Reply:
Nice point... I think there are only few g++ commands to be executed, not sure if it is because someone found it not helping too much... I simply didn't change...

Reply:
I just locally added an arbitrary "-j 5", by reading the log I think at most 3 jobs would be executed in parallel, which already agrees with what I found from the [Makefile](https://gitlab.com/hepcedar/professor/-/blob/main/Makefile?ref_type=heads) by a quick scan that not much targets being executed during `make`. Though I fully agree that we would benefit from building it in parallel indeed, at least a little bit.